server/apps/event/models.py

```diff
@@ -185,6 +185,12 @@ def clean_dates(values: dict, instance=None) -> OrderedDict:
         end_date = values.get('end_date', getattr(instance, 'end_date', None))
         if start_date and end_date and end_date < start_date:
             errors['end_date'] = gettext('Pick the end date later than start date. ')
+            return errors
+        crisis = values.get('crisis', getattr(instance, 'crisis', None))
+        if crisis.start_date and crisis.start_date > start_date:
```

Comment:
Just to be safe, limit `start_date` between crisis start and end.

```suggestion
        if start_date and crisis.start_date and crisis.start_date <= start_date <= crisis.end_date:
```
And apply these behaviors in other places.